CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ if(ENABLE_DATASOURCE)
   list(APPEND root_components_needed ROOTDataFrame)
 endif()
 find_package(ROOT ${root_min_version} REQUIRED COMPONENTS ${root_components_needed})
+find_package(Python3 REQUIRED COMPONENTS Development)
```

Comment:
```suggestion
```
This is unnecessary, since we already do that here:

https://github.com/AIDASoft/podio/blob/7ebfe4193f400c6852b82db53be3abdcc16bc6e9/cmake/podioBuild.cmake#L201-L204